rustls/src/msgs/handshake.rs

```diff
@@ -209,6 +209,41 @@ impl UnknownExtension {
     }
 }
 
+#[derive(Clone, Copy, Debug)]
+pub(crate) struct SupportedEcPointFormats {
+    pub(crate) uncompressed: bool,
+}
+
+impl Codec<'_> for SupportedEcPointFormats {
+    fn encode(&self, bytes: &mut Vec<u8>) {
+        let inner = LengthPrefixedBuffer::new(ECPointFormat::SIZE_LEN, bytes);
+
+        if self.uncompressed {
+            ECPointFormat::Uncompressed.encode(inner.buf);
+        }
+    }
+
+    fn read(r: &mut Reader<'_>) -> Result<Self, InvalidMessage> {
+        let mut spf = Self {
+            uncompressed: false,
+        };
+
```

Comment:
Tiny nit: consider just initializing a local `uncompressed` here and only initializing `Self { uncompressed }` in the return value expression?